.gitignore

```diff
@@ -2,14 +2,31 @@
 *~
 .*.sw[op]
 *.egg-info
+*.dist-info
+.eggs
+.coverage
+.coverage.*
+coverage.xml
 dist
 build
+logs
 _build
-local
 distribute-*
-.coverage
+docs/env
+local
+*venv*
+.idea
+Pipfile*
 .tox
+node_modules
+gtk/gi
+gtk/pygtkcompat
+gtk/cairo
```

Comment:
Most of these extra .gitignores make sense... but not these 4. Where are node_modules and a gtk directory coming from? The `bin` and `lib` folders below are also odd, but I guess they make more sense from a "general directory that might exist" sense.

Reply:
To be honest, I just copied over Toga's .gitignore.